gulpfile.js

```diff
@@ -544,13 +589,14 @@ function listAll(s3, bucketName, prefix, files, marker) {
     });
 }
 
-gulp.task('deploy-set-version', function() {
+gulp.task('deploy-set-version', function(done) {
     var buildVersion = yargs.argv.buildVersion;
     if (buildVersion) {
         // NPM versions can only contain alphanumeric and hyphen characters
         packageJson.version += '-' + buildVersion.replace(/[^[0-9A-Za-z-]/g, '');
         fs.writeFileSync('package.json', JSON.stringify(packageJson, undefined, 2));
     }
+    done();
```

Comment:
Why does this need the `done` callback?  It doesn't look like anything is async

Reply:
gulp 4 dropped support for synchronous tasks, so you have to take a done callback (or return a stream or promise), you can't just "do nothing".  More details here: https://gulpjs.com/docs/en/getting-started/async-completion